COMPONENTS

```diff
@@ -18,4 +19,5 @@ crtm        | v2.3.0            | 2.3.0
 nceppost    | dceca26           | dceca26
 wrf_io      | v1.1.1-cmake      | 1.1.1
 bufr        | bufr_v11.3.1.1    | 11.3.1.1
+wgrib2      | feature/cmake     | 2.0.8
```

Comment:
Do you want to create a branch say `v2.0.8-cmake` instead of `feature/cmake`

Reply:
Yes, I will do that

Reply:
you can merge after that change. thanks!